source/tutorials/hpc.rst

```diff
@@ -0,0 +1,688 @@
+.. _hpc:
+
+A Simple HPC Cluster on |CL|
+############################
+
+This tutorial demonstrates how to set a simple :abbr:`HPC (High
+Performance Computing)` cluster using `slurm`_, `munge`_, and 
+`PDSH`_.  For this tutorial, this cluster consists of a controller node 
+and four worker nodes, as shown in Figure 1.  For the sake of simplicity,
+each node resides on a separate host and their hostnames are hpc-controller,
+hpc-worker1, hpc-worker2, hpc-worker3, and hpc-worker4.  
+
+.. rst-class:: dropshadow
+
+.. figure:: /_figures/hpc/hpc-01.png
+   :alt: Simple HPC cluster
+
+   Figure 1: Simple HPC cluster
+
+The configuration is intentionally kept simple, notably avoiding setting 
+up cgroups and accounting. These and many more additional configuration 
+options can be added later.
+
+Prerequisites
+*************
+
+* Knowledge and experience with HPC 
+* Familiarity with slurm, munge, and PDSH
+* All nodes have synchronized clocks (typically by NTP)
+
+.. contents::
+    :local:
+    :depth: 1
+
+Set up controller node
+**********************
+
+In this step, install the cluster tools, enable the slurm controller service, 
+and configure and enable the the munge service.
+
+#. Install |CL| on the controller node and add a user with adminstrator 
+   privilege.  
+
+#. Boot it up and log in.
+
+#. Update |CL| to the latest version.
+
+   .. code-block:: bash
+
+      sudo swupd update
+
+#. Set the password for the root user.
+
+   .. code-block:: bash
+
+      sudo passwd root
+
+#. Set the hostname of the system to `hpc-controller`.
```

Comment:
Changing the hostname can have potentially negative effects. For example, if changing hostname remotely, this might cause problems with `sshd.service` and `NetworkManager-wait-online.service`, which could disable the connection. 

Please add some type of warning here--doesn't have to be specific as above--but users should understand this risk before executing commands.

Reply:
Can you point me to the source of this information?  hostnamectl is mostly setting the "pretty" name and shouldn't affect routability.  Perhaps, this step can be omitted to avoid confusion.

Reply:
The source is my tests. Will set up a call to discuss.